src/Templates/DotnetCore/Microsoft.NetMaui.CSharp.ProjectTemplates/MauiBlazor.vstemplate

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
+  <TemplateData>
+    <Name>.NET MAUI Blazor Hybrid App</Name>
```

Comment:
Looks like the other templates are using the package and id for the name?

Reply:
Some do, but some also have hard-coded names list this, see https://github.com/dotnet/templates/blob/af41e3b78267762967ea0f155a8641a5cffc2187/src/Templates/DotnetCore/Microsoft.NetCore.VB.ProjectTemplates.Desktop/Winforms.vstemplate#L4C9-L4C9

Similar to the comment about the description, the name doesn't matter, it just needs a valid value.